Jenkinsfile

```diff
@@ -10,90 +10,105 @@ pipeline {
     }
     environment {
         DOCS = 'distribution/docs'
-        ITESTS= 'distribution/test/itests/test-itests-ddf'
+        ITESTS = 'distribution/test/itests/test-itests-ddf'
+        POMFIX = 'libs/libs-pomfix,libs/libs-pomfix-run'
+        LARGE_MVN_OPTS = '-Xmx8192M -Xss128M -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC '
```

Comment:
✏️  might be worth a comment as to why we have some of these values for people who may contribute to the file later.